### PR TITLE
ML-043/add movie award

### PR DESCRIPTION
**Pull request by WhatWasThatt**
# Description

add movie award(model, migration, serializer, policy, controller, route, validation, index, seeds)
add movie award specs(factory, model, request, routing, policy)
add association with person
add category(model, migration, serializer, policy, controller, route, validation, seeds)
add category specs(factory, model, request, routing, policy)
update seeds

## Trello card link

https://trello.com/c/dqT7wdV1

# Checklist:

- [x] My code fixes the problem or adds required functionality
- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] My changes generate no new warnings
- [x] I have added tests that prove that my feature works
- [x] New and existing unit tests pass locally with my changes

# Additional requirements:

- [x] Does your code require any migrations to be ran?
- [ ] Does your code require installing any new gems or system dependencies?
- [ ] Does your code require any changes to current deployment process?

# Additional Information

### CODE REVIEW

File: spec/models/award_spec.rb [hunk 1]
```diff
@@ -50,5 +50,6 @@
 
   context 'with associations' do
     it { should belong_to(:country) }
+    it { should have_many(:categories) }
```
Comment (twill11) [inline spec/models/award_spec.rb:53]: can you replace please should with is_expected, for all associations, thanks!
image

Comment (WhatWasThatt) [reply spec/models/award_spec.rb:53]: https://github.com/thoughtbot/shoulda-matchers

Comment (twill11) [reply spec/models/award_spec.rb:53]: ok than!

**Review by twill11 — APPROVED** [review]
GJ!

File: spec/models/award_spec.rb (hunk 1 above)
Comment (WhatWasThatt) [reply spec/models/award_spec.rb:53]: my bad

**Review by twill11 — APPROVED** [review]
Good job! Thanks for your changes

**Review by dmitrymalllpr — APPROVED** [review]

**Review by KarinaKlevzhits — APPROVED** [review]

**Pull request merged**